### PR TITLE
Update libsingular to 0.10 for Julia 1.3

### PR DESCRIPTION
**Pull request by tthsqe12**
@thofma

### CODE REVIEW

**Review by thofma — APPROVED** [review]

File: L/libsingular_julia/common.jl [hunk 1]
```diff
@@ -5,11 +5,11 @@ import Pkg: PackageSpec
 import Pkg.Types: VersionSpec
 
 name = "libsingular_julia"
-version = VersionNumber(0, 9, 200 + julia_version.minor)
+version = VersionNumber(0, 10, julia_version.minor)
```
Comment (giordano) [inline L/libsingular_julia/common.jl:8]: Is removing the 200 correct?  Sorry, I don't follow the versioning scheme of the package :slightly_smiling_face:

Comment (thofma) [reply L/libsingular_julia/common.jl:8]: To simuate patch number this was introduced here: https://github.com/fingolfin/Yggdrasil/commit/850197a6ec28b4c28ddc0ba06fb621cdbf84db30#diff-92b56ff034ba8b37e5e99a14e623b39088eaad34e0f223c963c24b885f3c3252

**Comment by thofma** [comment]
wait with the merge plase

**Comment by giordano** [comment]
Oops, too late, but I cancelled the CI job on `master`

**Comment by thofma** [comment]
We had a minor discussion about `9.300 +` or `0.10.0 +`. We decided that the patch here is the right one. How should we proceed? Sorry for the hassle.

**Comment by tthsqe12** [comment]
version 0.10.0 is fine

File: L/libsingular_julia/common.jl (hunk 1 above)
Comment (tthsqe12) [reply L/libsingular_julia/common.jl:8]: we will go with 0.10.0

**Comment by giordano** [comment]
I'll restart the job

**Comment by thofma** [comment]
Thanks!

**Pull request merged**